src/css/components/_control.scss

```diff
@@ -36,3 +36,28 @@
   display: table-cell;
   vertical-align: middle;
 }
+
+// visually hide a control-bar component without changing the position
+// and without hiding it from screen readers. This also preserves
+// a transition animation.
+.video-js .vjs-control .vjs-visual-hide-vertical {
```

Comment:
is it necessary to have two of these? Why not just set both the height and width to be `1px` in all cases?
What about other visual hiding techniques like `left: -9999px;` or the recommended approach from webaim: http://webaim.org/techniques/css/invisiblecontent/#absolutepositioning?